multimodal/src/autogluon/multimodal/optimization/utils.py

```diff
@@ -99,7 +97,11 @@ def get_loss_func(
         if mixup_active:
             loss_func = SoftTargetCrossEntropy()
         else:
-            loss_func = nn.CrossEntropyLoss()
+            if loss_func_name is not None and loss_func_name.lower() == "focal_loss":
+                alpha, gamma, reduction = get_focal_loss_params(loss_config)
+                loss_func = FocalLoss(alpha=alpha, gamma=gamma, reduction=reduction)
```

Comment:
`get_focal_loss_params() ` seems unnecessary. A more concise way may be like this:

```
loss_func = FocalLoss(
    alpha=OmegaConf.select(config, "focal_loss.alpha"),
    gamma=OmegaConf.select(config, "focal_loss.gamma"),
    reduction=OmegaConf.select(config, "focal_loss.reduction"),
)
```

Reply:
oh so if the `focal_loss` is written in the config yaml file, it will be found by OmegaConf.select no matter what the user inputs?

I added the sanity checks just in case users delete the `focal_loss` attribute in the config yaml.

Reply:
I think OmegaConf.select can handle the missing keys.